en/ros/ros2_comm.md

```diff
@@ -428,6 +428,35 @@ Similarly to vectors, also quanternions representing the attitude of the vehicle
 
 [PX4/px4_ros_com](https://github.com/PX4/px4_ros_com) provides the shared library [frame_transforms](https://github.com/PX4/px4_ros_com/blob/main/include/px4_ros_com/frame_transforms.h) to easily perform such conversions.
 
+### ROS, Gazebo and PX4 time synchronization
+
+By default, time synchronization between ROS 2 and PX4 is automatically managed by the [uXRCE-DDS middleware](https://micro-xrce-dds.docs.eprosima.com/en/latest/time_sync.html) and time synchronization statistics are available listening to the bridged topic `/fmu/out/timesync_status`.
+When the uXRCE-DDS client runs on a flight controller and the agent runs on a companion computer this is the desired behavior as time offsets, time drift, and communication latency, are computed and automatically compensated.
+
+For Gazebo simulations PX4 uses the Gazebo `/clock` topic as [time source](../sim_gazebo_gz/README.md#px4-gazebo-time-synchronization) instead.
+This clock is always slightly off-sync w.r.t. the OS clock (the real time factor is never exactly one) and it can can even run much faster or much slower depending on the [user preferences](http://sdformat.org/spec?elem=physics&ver=1.9).
+Note that this is different from the [simulation lockstep](../simulation/README.md#lockstep-simulation) procedure adopted with Gazebo Classic.
+
+ROS2 users have then two possibilities regarding the [time source](https://design.ros2.org/articles/clock_and_time.html) of their nodes.
+
+#### ROS2 nodes use the OS clock as time source
+
+This scenario, which is the one considered in this page and in the [offboard_control](./offboard_control.md) guide, is also the standard behavior of the ROS2 nodes.
+The OS clock acts as time source and therefore it can be used only when the simulation real time factor is very close to one.
+The time synchronizer of the uXRCE-DDS client then bridges the OS clock on the ROS2 side with the Gazebo clock on the PX4 side.
+No further action is required by the user.
+
+#### ROS2 nodes use the Gazebo clock as time source
+
+In this scenario, ROS2 also uses the Gazebo `/clock` topic as time source.
+This approach makes sense if the Gazebo simulation is running with real time factor different from one, or if ROS2 needs to directly interact with Gazebo.
+The procedure requires you to:
+
+- Directly bridge ROS and Gazebo using [ros_gz](https://github.com/gazebosim/ros_gz) and the [ros_gz_bridge](https://github.com/gazebosim/ros_gz) package.
```

Comment:
@beniaminopozzan How?

Otherwise this looks very clear. I'll merge and you can update if you don't think this is obvious (I don't, unless the following instructions are what do this, in which case I'd rewrite slightly).

Reply:
> you can update if you don't think this is obvious (I don't, unless the following instructions are what do this, in which case I'd rewrite slightly)

Thanks @hamishwillee, You are right, let me rephase it in another PR